lib-utilities/etc/Base.1.10.0.json

```diff
@@ -1,7 +1,7 @@
 {
     "@Redfish.Copyright": "Copyright 2014-2021 DMTF. All rights reserved.",
     "@Redfish.License": "Creative Commons Attribution 4.0 License.  For full text see link: https://creativecommons.org/licenses/by/4.0/",
-    "@odata.type": "#MessageRegistry.v1_4_0.MessageRegistry",
+    "@odata.type": "#MessageRegistry.v1_4_2.MessageRegistry",
```

Comment:
Please donot change anything in the json files. these files should come from redfish spec. update the json to latest. applies for all the json's

Reply:
Made the required changes